### PR TITLE
Added api1800 support to network set and uplinkset

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added api1800 support to network set and uplinkset

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api1800/synergy/network_set.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1600/c7000/network_set'
+
+module OneviewSDK
+  module API1800
+    module Synergy
+      # Network set resource implementation for API1800 Synergy
+      class NetworkSet < OneviewSDK::API1600::C7000::NetworkSet
```
Comment (chebroluharika) [inline lib/oneview-sdk/resource/api1800/synergy/network_set.rb:18]: This should be API1800::C7000. Please change accordingly in API1200 also.

File: spec/unit/resource/api1800/synergy/network_set_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1800::Synergy::NetworkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1600::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API1600::C7000::NetworkSet
```
Comment (chebroluharika) [inline spec/unit/resource/api1800/synergy/network_set_spec.rb:18]: It should be API1800::C7000

File: lib/oneview-sdk/resource/api1800/synergy/network_set.rb (hunk 1 above)
Comment (VenkateshRavula) [reply lib/oneview-sdk/resource/api1800/synergy/network_set.rb]: done

File: spec/unit/resource/api1800/synergy/network_set_spec.rb (hunk 2 above)
Comment (VenkateshRavula) [reply spec/unit/resource/api1800/synergy/network_set_spec.rb]: done

**Pull request merged**